default.yml

```diff
@@ -103,14 +91,17 @@ Style/StringLiterals:
 Style/HashSyntax:
   EnforcedStyle: ruby19_no_mixed_keys
 
+# https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashEachMethods
 Style/HashEachMethods:
-  Enabled: false
+  Enabled: true
```

Comment:
I think it makes sense to prefer `each_key` when possible:

```ruby
# disabled
hash.keys.each { |k| p k }
hash.values.each { |v| p v }

# enabled
hash.each_key { |k| p k }
hash.each_value { |v| p v }
```

Reply:
I would need more data on this one, is there a performance gain here?

Reply:
I guess there could be a perf benefit I didn't know about!

https://stackoverflow.com/a/46533443/398696

But I mostly think it looks nicer :p

Reply:
🆗

Reply:
I'm good with this